vassal-app/src/main/java/VASSAL/launch/DummyConfigImpl.java

```diff
@@ -0,0 +1,45 @@
+package VASSAL.launch;
+
+import java.nio.file.Path;
+
+public class DummyConfigImpl implements Config {
```

Comment:
Newly steeped in the ideology that "Impl"-is-always-a-bad-name, I would suggest naming this simply "DummyConfig". Sort of like ArrayList is named ArrayList instead of ArrayListImpl :)